ci/Dockerfile

```diff
@@ -37,6 +37,6 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86
     && conda config --set solver libmamba
 
 # install cuML
-ARG CUML_VER=24.08
+ARG CUML_VER=24.10
 RUN conda install -y -c rapidsai-nightly -c conda-forge -c nvidia cuml=$CUML_VER python=3.9 cuda-version=11.8 \
```

Comment:
24.08 is moved from "nightly" to "stable" release. 24.10 is under development for the next release so can be unstable. See rapids install selector for more info: https://docs.rapids.ai/install
 
Could you keep "24.08" and replace "rapidsai-nightly" with "rapidsai"?

Reply:
Yes lets keep 24.10 on 24.08 rapidsai until it stabilizes a bit more and then switch to nightly.
But if nightly 24.08 is gone, can you also change this line to 24.08 rapidsai (i.e. not nightly) in the 24.08 branch?
https://github.com/NVIDIA/spark-rapids-ml/blob/branch-24.08/ci/Dockerfile#L41

Reply:
Sounds good to me

BTW, this change targeted to spark-rapids-ml branch-24.10